### PR TITLE
HOUSNAV-39: 9.9.9.json terms update

### PR DESCRIPTION
**Pull request by wardm89**
[HOUSNAV-39](https://hous-bssb.atlassian.net/browse/HOUSNAV-39)

## Overview & Purpose

Update some of the 9.9.9 questions to include missing links for defined term variations and add 2 more terms

## Testing

Navigate to the term referenced in the ticket's comment and verify the tooltip works and definition is opened when clicked

## Checklist
- [x] I have written or updated vitests for this work
- [x] I have reviewed the [accessibility guidelines](https://digital.gov.bc.ca/wcag/home/) relevant to this work
- [x] I have reviewed this work with at least one screen reader (when applicable)
- [x] I have added/updated any related documentation

### CODE REVIEW

**Comment by laurenrevere** [comment]
Everything you updated looks good.


There are a couple more updates that I think would be valuable to add here. Documented on Jira: https://hous-bssb.atlassian.net/browse/HOUSNAV-39?focusedCommentId=11886

**Comment by wardm89** [comment]
> Everything you updated looks good.
> 
> There are a couple more updates that I think would be valuable to add here. Documented on Jira: https://hous-bssb.atlassian.net/browse/HOUSNAV-39?focusedCommentId=11886

I just pushed the updates. VERY good catches. Also, I did update the focus functionality. Essentially if you clicked the same term twice it didn't change the "focusSection" so it wasn't triggering the view to change. That's updated now so that if you click the same term back to back after scrolling it pops the definition back into view.

Also, "Group F" isn't actually a group with a definition so I combined the 3 divisions for that and made it it's own hidden term. Let me know if you think that's ok.

**Comment by laurenrevere** [comment]
Screenshot 2024-07-05 at 9 30 57 AM

Floor area has defined term link within its tooltip.

**Comment by laurenrevere** [comment]
> > Everything you updated looks good.
> > There are a couple more updates that I think would be valuable to add here. Documented on Jira: https://hous-bssb.atlassian.net/browse/HOUSNAV-39?focusedCommentId=11886
> 
> I just pushed the updates. VERY good catches. Also, I did update the focus functionality. Essentially if you clicked the same term twice it didn't change the "focusSection" so it wasn't triggering the view to change. That's updated now so that if you click the same term back to back after scrolling it pops the definition back into view.
> 
> Also, "Group F" isn't actually a group with a definition so I combined the 3 divisions for that and made it it's own hidden term. Let me know if you think that's ok.

Thinking about Group F, would it make more sense to just say "industrial occupancies" since it seems to cover all of them?

Also, I know I added it to my follow up ticket and didn't add it to the notes for this ticket, but can we remove the repeated "Industrial occupancy" in this definition?
Screenshot 2024-07-05 at 9 37 33 AM

File: packages/data/json/terms.json [hunk 1]
```diff
@@ -152,7 +163,7 @@
     "cleanDefinition": "Mercantile occupancies means the occupancy or use of a building or part thereof for the displaying or selling of retail goods, wares or merchandise."
   },
   "noncombustible": {
-    "definition": "<glossary-term>Noncombustible</glossary-term> means that a material meets the acceptance criteria of CAN/ULC-S114, Standard Method of Test for Determination of Non-Combustibility in <defined-term-glossary>building</defined-term-glossary> materials.",
+    "definition": "<glossary-term>Noncombustible</glossary-term> means that a material meets the acceptance criteria of CAN/ULC-S114, \"Standard Method of Test for Determination of Non-Combustibility in Building materials.\"",
```
Comment (laurenrevere) [inline packages/data/json/terms.json]: "Materials" should be capitalized here.

**Review by laurenrevere — APPROVED** [review]
Looks good!

**Pull request merged**